Interop/Runtime/src/native/kotlin/kotlinx/cinterop/ObjectiveCImpl.kt

```diff
@@ -132,6 +132,10 @@ internal annotation class ObjCMethodImp(val selector: String, val encoding: Stri
 @TypedIntrinsic(IntrinsicType.OBJC_GET_SELECTOR)
 internal external fun objCGetSelector(selector: String): COpaquePointer
 
+@PublishedApi
+@TypedIntrinsic(IntrinsicType.OBJC_INSTANCE_OF)
+internal external inline fun <reified T : ObjCObjectBase> objCInstanceOf(obj: Any): Boolean
```

Comment:
```suggestion
internal external inline fun <reified T : ObjCObject> objCInstanceOf(obj: Any): Boolean
```
(would probably be more correct here)